wallet/src/main/java/com/android/identity_credential/wallet/ui/prompt/passphrase/PassphraseEntryField.kt

```diff
@@ -221,7 +222,7 @@ private fun calcHintAndMeetsRequirements(
         )
     }
 
-    if (isWeakPassphrase(passphrase)) {
+    if (checkWeakPassphrase && isWeakPassphrase(passphrase)) {
```

Comment:
I think this is just an optimization.

Reply:
It is not: if user enters a PIN which is weak (but for whatever reason is a valid PIN for an eID card), meetsRequirements will never be true